openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -633,8 +639,8 @@ def run(self, *, dry=False, verbose=True,
             return {
                 'system_pdb': shared_basepath / output_settings.preminimized_structure,
                 'minimized_pdb': shared_basepath / output_settings.minimized_structure,
-                'nvt_equil_pdb': shared_basepath / output_settings.equil_NVT_structure,
-                'npt_equil_pdb': shared_basepath / output_settings.equil_NPT_structure,
+                'nvt_equil_pdb': shared_basepath / output_settings.equil_nvt_structure,
```

Comment:
this file isn't always created any more, so probably need to only add this dict entry if the file is created

Reply:
I also made the `equil_npt_structure` optional, so will also change it for that. Is there a way of doing that that is not a series of if statements?

Reply:
I don't think so